imagevector/images.yaml

```
@@ -107,7 +107,6 @@ images:
   sourceRepository: github.com/kubernetes/cloud-provider-gcp
   repository: registry.k8s.io/cloud-provider-gcp/cloud-controller-manager
   # TODO(AndreasBurger,kon-angelo): Update to v31.x.y image of the cloud-controller-manager when it is available.
-  # TODO(LucaBernstein,AndreasBurger,kon-angelo): Add entry for v32.x.y image of the cloud-controller-manager when it's available.
   tag: "v30.0.0"
   targetVersion: ">= 1.31"
```

```suggestion
  targetVersion: "1.31.x"
```

Otherwise it won't work the way you expect. :) 

When you have 

```yaml
  repository: registry.k8s.io/cloud-provider-gcp/cloud-controller-manager
  tag: "v30.0.0"
  targetVersion: ">= 1.31"
```

and 

```
  repository: registry.k8s.io/cloud-provider-gcp/cloud-controller-manager
  tag: "v32.2.2"
  targetVersion: ">= 1.32"
```

which version it should pick up for K8s 1.31?